### ISSUE TITLE
Custom resolution invalid in game version 6.6

### ISSUE THREAD

**Issue by PandaAdap** [BuIlDaLiBlE/BetterHI3Launcher#67]
**Describe the bug**
Custom resolution invalid in game version 6.6, but the custom FPS function still work. Launcher version is v1.4.20230329.0. 

**To Reproduce**
1. Go to 'Option->Manage game settings->Set custom resolution'.
2. Set a resolution. (My resolution is 3440 x 1440)
3. Click on 'Confirm', then prompt for successful set.
4. Log output 'Setting game resolution to 3440x1440, fullscreen enabled...success!'
5. Then click ‘Launch’, but custom resolution is invalid, it set to default 2560x1440.
6. Go to 'Option->Manage game settings->Set custom resolution' again, the width and height have been reset to 2560x1440.

**Expected behavior**
The game resolution should be set to 3440x1440 and will not be reset.

**Log**
BetterHI3Launcher v1.4.20230329.0
Working directory: G:\\Honkai Impact 3\\
OS version: Windows 10 Pro for Workstations (Version 22H2, Build 19045.2846)
OS language: zh-CN
Launcher language: en
WARNING: Bp Network connection error, attempt №2...
Using server: CN
Using mirror: HoYoverse
Checking for game update...
Game version: 6.6.0
Game directory: G:\\Honkai Impact 3\\Games
Setting game resolution to 3440x1440, fullscreen enabled...success!

**Comment by BuIlDaLiBlE** [comment]
Custom resolution can only be used in windowed mode, uncheck the fullscreen checkbox and try again.

**Comment by PandaAdap** [comment]
I tryed just now. It works while in startup screen, but once entering the loading screen, it turns to 2560x1440 again.
---
Startup screen:
![Startup](https://user-images.githubusercontent.com/37203937/233791515-1441e1f9-a5f6-454e-aa5a-18aa8c40ff3f.png)
---
Loading screen:
![loading](https://user-images.githubusercontent.com/37203937/233791523-4e6102ab-adfe-44a2-9d6d-4aeb6923b785.png)
---
I checked the fullscreen before v6.6 update, and it works.
Check this, pls.

Thanks!
